src/include/storage/record_buffer.h

```diff
@@ -3,15 +3,15 @@
 #include "common/constants.h"
 #include "common/object_pool.h"
 #include "common/typedefs.h"
-#include "storage/delta_record.h"
+#include "storage/undo_record.h"
 
 namespace terrier::storage {
 
 /**
  * An UndoBufferSegment is a piece of (reusable) memory used to hold undo records. The segment internally keeps track
```

Comment:
I think the name UndoBufferSegment should be changed to BufferSegment since both RedoBuffer and UndoBuffer use this.